examples/classification/convert_from_original/convert_vgg.py

```diff
@@ -0,0 +1,37 @@
+import chainer
```

Comment:
> Putting a model-specific utility under task-specific directory seems against your intention. The conversion code for VGG-16 can be used for other task than classification.

OK. Your suggestion seems easier for users who are not interested in classification to access the conversion code.

> If you want to reduce the number of subdirectories, how about putting detection models under detection?

The current layout of directories are easier to explore than the nested directories.
I think it is still manageable.
If things start to get out of control, we can write a README under `examples/`.